pages/journal/[id].tsx

```diff
@@ -32,7 +32,8 @@ const JournalPostPage: NextPage<Props> = ({ post, relatedEntries }) => {
     return (
         <main className="container">
             <Head>
-                <title>Journal | MindVersity</title>
+                <title> {post.title} | Journal | MindVersity - A peer mental health network.</title>
+                <meta name="description" content={`${post.description ? post.description : "A journal entry at MindVersity."}`}></meta>
```

Comment:
Do we have a length limit on journal descriptions? If not, we might need to concatenate this title and meta description as well.

Reply:
I don't think there's a length limit. You're right, it probably would be best to concatenate it as well.